Workbooks/WorkbooksMetadata.json

```diff
@@ -1415,5 +1415,18 @@
       "templateRelativePath": "AzureSentinelCost.json",
       "subtitle": "",
       "provider": "Azure Sentinel Community"
+    },   
+    {
+      "workbookKey": "AzurePurview",
+      "logoFileName": "AzurePurview.svg",
+      "description": "This workbook provides insights into the sensitivity of data that has been scanned by Azure Purview, including classifications and labels found across the data estate.",
+      "dataTypesDependencies": ["PurviewDataSensitivityLogs"],
+      "dataConnectorsDependencies": ["AzurePurview"],
```

Comment:
DataConnectorDependencies should have the value of  **id** attribute from DataConnector json file.

Reply:
DataConnectorDependencies has been updated to have the id attribute for the data connector.